app/views/taxons/_taxonomy_tree.html.erb

```diff
@@ -1,22 +1,52 @@
 <div class="taxonomy-tree">
-  <% taxonomy_tree.each do |tree_node| %>
-    <div class="taxon-level taxon-depth-<%= tree_node.node_depth %>">
-      <span class="taxon-level-title">
-        <%= link_to tree_node.title, taxon_path(tree_node.content_id) %>
-      </span>
+  <div class="taxon-parents">
+    <% taxonomy_tree.immediate_parents.each do |parent_node| %>
+      <div class="parent-expansion">
+        <div class="taxon-level taxon-depth-1">
+          <span class="taxon-level-title">
+            <%= link_to parent_node.title, taxon_path(parent_node.content_id) %>
+          </span>
+        </div>
+        <% parent_node.each do |parent_ancestor| %>
+          <% next if parent_ancestor.node_depth == 1 %>
+          <div class="taxon-level taxon-depth-<%= parent_ancestor.node_depth %>">
+            <span class="taxon-level-title">
+              <%= link_to parent_ancestor.title, taxon_path(parent_ancestor.content_id) %>
+            </span>
+          </div>
+        <% end %>
+      </div>
+    <% end %>
+  </div>
 
-      <% parent_taxons = tree_node.parent_taxons %>
+  <% parents_class = taxonomy_tree.immediate_parents.present? ? "taxon-focus--has-parents" : "" %>
```

Comment:
It's a reasonably common CSS convention to place modifiers after a `--`, see SUIT CSS and BEM, for example. No mention of anything like this in the GOV.UK CSS styleguide so I figured it's legit.